code/aggregation/aggregate-raw-data.R

```diff
@@ -368,26 +368,12 @@ all.data$Phoneme <- denorm(all.data$Phoneme)
 all.data$Allophones <- denorm(all.data$Allophones)
 # FACTOR TO DROP UNUSED LEVELS
 all.data$Phoneme <- factor(all.data$Phoneme)
+# MARK MARGINAL PHONEMES AND REMOVE ANGLE BRACKETS
+all.data <- markMarginal(all.data)
```

Comment:
this (and the big deletion below it) are changes merged in from upstream. They are "new to you" because bambooforest/master branch has never seen them before (they were merged from drammock/fix-feature-merging into phoible/master in [#74](https://github.com/phoible/phoible/pull/74) without bambooforest/master ever touching them).